NitroxModel/Packets/DroppedItemPacket.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using NitroxModel.DataStructures.Util;
+using UnityEngine;
+
+namespace NitroxModel.Packets
+{
+    [Serializable]
+    public class DroppedItemPacket : Packet
```

Comment:
Please keep the name as "DroppedItem" as we do not have the word "Packet" in our packet classes.